drivers/bluetooth/hci/rpmsg.c

```diff
@@ -294,6 +298,10 @@ static int bt_rpmsg_open(void)
 	const struct device *hci_ipc_instance =
 		DEVICE_DT_GET(DT_CHOSEN(zephyr_bt_hci_rpmsg_ipc));
 
+#ifdef CONFIG_SOC_NRF5340_CPUAPP
+	cpunet_forceon();
```

Comment:
Please add weak functions instead of the Kconfig `#ifdef`:
See here:
https://github.com/zephyrproject-rtos/zephyr/blob/29bbcb1e38f84b505c7fe7e3f7ab4bb3497eea4c/drivers/bluetooth/hci/h4.c#L498

Also, the name needs to change. Please use the same one as above, `bt_hci_transport_setup`.

Reply:
@carlescufi Both comments addressed.